logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/register/RequestActivationCodeUseCase.kt

```diff
@@ -0,0 +1,26 @@
+package com.wire.kalium.logic.feature.register
+
+import com.wire.kalium.logic.NetworkFailure
+import com.wire.kalium.logic.configuration.ServerConfig
+import com.wire.kalium.logic.data.register.RegisterAccountRepository
+
+class RequestActivationCodeUseCase(
+    private val registerAccountRepository: RegisterAccountRepository
+) {
+    suspend operator fun invoke(email: String, serverConfig: ServerConfig): RequestActivationCodeResult {
+
+        return registerAccountRepository.requestEmailActivationCode(email, serverConfig.apiBaseUrl)
+            .fold({
+                RequestActivationCodeResult.Failure.Generic(it)
+            }, {
+                RequestActivationCodeResult.Success
+            })
+    }
+}
```

Comment:
```suggestion
    suspend operator fun invoke(email: String, serverConfig: ServerConfig) = registerAccountRepository.
   requestEmailActivationCode(email, serverConfig.apiBaseUrl)
        .fold({
            RequestActivationCodeResult.Failure.Generic(it)
        }, {
            RequestActivationCodeResult.Success
        })
}
```

what do you think of ignoring body definition for function with one return?